django/utils/translation/trans_real.py

```diff
@@ -412,11 +412,12 @@ def get_languages():
 @functools.lru_cache(maxsize=1000)
 def get_supported_language_variant(lang_code, strict=False):
     """
-    Return the language-code that's listed in supported languages, possibly
+    Return the language code that's listed in supported languages, possibly
```

Comment:
Removing the `-` seems fine. (5 uses of `language-code` vs 55 of `language code`.)